src/vm/mscorlib.h

```diff
@@ -615,17 +615,6 @@ DEFINE_CLASS(OLE_AUT_BINDER,        System,                 OleAutBinder)
 DEFINE_CLASS(MONITOR,               Threading,              Monitor)
 DEFINE_METHOD(MONITOR,              ENTER,                  Enter,                      SM_Obj_RetVoid)
 
-DEFINE_CLASS_U(Threading,              OverlappedData, OverlappedDataObject)
```

Comment:
You should keep these `_U` entries. They validate that the managed layout is in sync with the unmanaged layout.

Reply:
I forgot to remove the whole unmanaged definition apparently.